release-engineering/OWNERS

```diff
@@ -3,6 +3,8 @@
 approvers:
   - aleksandra-malinowska
   - calebamiles
+  - ixdy
+  - mikedanese
```

Comment:
Can you mention in the commit message / PR description the rationale for their addition.  They seem like fine folks to add, but I don't know if they volunteered or were voluntold or something else.

Reply:
The intention here was to sync the current [OWNERS of k/release](https://github.com/kubernetes/release/blob/master/OWNERS_ALIASES#L8-L13) with the subproject OWNERS.

This is mostly to bolster OWNERS, but release engineering should feel free to update/remove as appropriate once this merges.

Updated the commit message to reflect.